### ISSUE TITLE
[Suggestion] Add raw ores to crushable items

### ISSUE THREAD

**Issue by Mantarri** [Ellpeck/ActuallyAdditions#1412]
Pre-raw ore items becoming a thing, the crusher could be used to double your ore yield once you had the resources to make it. The crusher hasn't had the raw ore items added to the list of items it can crush, so now you'd need to get silk touch before being able to ore double. Perhaps this is an intended balance shift, but if not, it really pushes when you're able to ore double further into progression than it used to be.

If this is intentional, is there a way to add crusher recipes via datapack, so I can add them for my own playthrough? I tried looking through the repo, and from what I could see, the crusher recipes are only addable using Java, which is a bit cumbersome.

**Comment by Antonino545** [comment]
News?

**Comment by RippleRose** [comment]
i would also like to see the addition of datapack support for custom crusher and Atomic Reconstructor if possible <3

**Comment by Flanks255** [comment]
Both of those examples are datapack driven, and only datapack driven, the old java api calls to add recipes dont connect to anything anymore, and have been slowly being removed.

[Crushing Recipes](https://github.com/Ellpeck/ActuallyAdditions/tree/main/src/generated/resources/data/actuallyadditions/recipe/crushing)
[Atomic Reconstructor recipes](https://github.com/Ellpeck/ActuallyAdditions/tree/main/src/generated/resources/data/actuallyadditions/recipe/laser)

**Comment by RippleRose** [comment]
yeah... i realised after poking through your source code and seeing EVERYTHING commented out haha

felt a bit foolish tbh ^.^

I appreciate the work you guys are all doing on the mod.

thank you for your hard work ♡
